src/collections/_documentation/performance/alerts-notifications/alerts.md

```diff
@@ -0,0 +1,192 @@
+
+### **Digests**
+
+The digests feature works only for **issue alert emails**. This project-level setting allows you to batch and deliver issue alert emails for a project. Use the sliders to control the frequency.
+
+![https://docs.sentry.io/assets/notifications/alert_digest-e0a488f304173985c095f8ef763a0d0f68d34336a00e114625cbb339d7e6f486.png](https://docs.sentry.io/assets/notifications/alert_digest-e0a488f304173985c095f8ef763a0d0f68d34336a00e114625cbb339d7e6f486.png)
+
+## Alert Listing
+
+This is a listing of all your project's alerts.
+
+![DON%20T%20add%20content%20to%20this%20Draft%20Alerts%20Notificatio/Untitled%201.png](DON%20T%20add%20content%20to%20this%20Draft%20Alerts%20Notificatio/Untitled%201.png)
+
+## Alert Subscription
+
+### Issue Owners
+
+By default, **alert emails** are sent to [LINK: issue owners]. If issue owners isn't configured, or an owner isn't found, the email will do one of two things:
+
+- not send
+- send to all project members depending on the fall-through setting under **Project Settings > Issue Owners**.
+
+![DON%20T%20add%20content%20to%20this%20Draft%20Alerts%20Notificatio/Screenshot_2020-02-26_22.03.07.png](DON%20T%20add%20content%20to%20this%20Draft%20Alerts%20Notificatio/Screenshot_2020-02-26_22.03.07.png)
+
+### Ignore
+
+While an issue is ignored, all alerts for that issue are muted. You can also ignore an issue until certain conditions are met — for example, "ignore for 30 minutes." Keep in mind; **an ignored issue will still count towards your quota**.
+
+Finally, you can configure your personal alert notifications settings [here](https://sentry.io/settings/account/notifications/).
+
+### Personal Alert Settings
+
+You can find your personal alert settings in **User Settings > Account > Notifications**.
+
+Use the "default project alerts" setting to set your default preference across all projects — subscribed or unsubscribed.
+
+![https://docs.sentry.io/assets/notifications/default_project_alerts-132a6776016d0827f14548f4e970adbcb273174f42c9aacfce2260e7c5434dfc.png](https://docs.sentry.io/assets/notifications/default_project_alerts-132a6776016d0827f14548f4e970adbcb273174f42c9aacfce2260e7c5434dfc.png)
+
+After making a selection, you can also selectively change it per project by going to **User Settings > Account > Notifications > Fine tune alerts by project**.
+
+Each project has three options: Default, On, or Off. Selecting default uses your default preference from the previous step.
+
+![https://docs.sentry.io/assets/notifications/specific_project_alert-8b39db12d437131adf61d367d1a7161bb6b7c5a99d2ead1c6e339aac98dc9e46.png](https://docs.sentry.io/assets/notifications/specific_project_alert-8b39db12d437131adf61d367d1a7161bb6b7c5a99d2ead1c6e339aac98dc9e46.png)
+
+## **FAQs**
+
+- Can I copy an alert rule to another project?
+    - Unfortunately, this is not an option.
```

Comment:
perhaps change to "Not in this release"? Do we have default wording we typically use here?

Reply:
We don't have default wording for something like this, but I think that's a great idea. How about. "Although this feature is currently unavailable, Sentry is developing improvements."